src/components/PopupDeleteCard.js

```diff
@@ -0,0 +1,21 @@
+import Popup from "./Popup.js";
+
+class PopupDeleteCard extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+    this._popupSubmitButton = this._popupElement.querySelector(
+      ".modal__button-delete"
+    );
+  }
+
+  setSubmitAction(callBack) {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.